cmd/cloud_login.go

```diff
@@ -30,7 +28,7 @@ func getCmdCloudLogin(gs *state.GlobalState) *cobra.Command {
 
 	// loginCloudCommand represents the 'cloud login' command
 	exampleText := getExampleText(gs, `
-  # Log in with an email/password.
+  # Prompt for a k6 Cloud token.
```

Comment:
Should we keep both: prompting for a token when running `k6 cloud login` and `-t [TOKEN]`?

It sounds a bit redundant, and as we're planning to ship it in v1.0, I think it deserves some care in terms of carefully thinking of its implications.

Some thoughts:
- In favor of `k6 cloud login`: 
  - Default behavior, user friendly walkthrough (although it could be replaced with a decent help)
  - Doesn't fully (only partially) break the currently existing, interactive, behavior.
- In favor of `-t [TOKEN]`:
  - Friendlier for CLI environments (less interactivity is required)
  - More flexible for future auth mechanisms (extensibility after v1.0)

Reply:
I would like to invest time in improving the interactive flow in the future (picking default stack, magic link to the browser to authenticate, etc). But yeah, no strong opinions on what we do with it as of today. It is true that if it is only to pass the token it feels a bit redundant.

Reply:
Then, I'd probably vote to keep both but generally recommend the use of `-t`, I guess 🤔

Reply:
Sorry for the delay, I agree, we should keep both: prompt for the token, and support `-t` 👍🏻